microsoftgraph/microsoft-graph-explorer-v4
# Remove scrollbars from expanded modal

## Overview
Fixes #1981 

### Demo
![image](https://user-images.githubusercontent.com/45680252/182645388-a23f04de-a84e-4ce6-bd1c-fd07d52c8a0c.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Ensure there is a url on the url textbox
Click on the Expand button
On the expanded modal, click on the Code snippets tab
Observe fix
